airbyte-integrations/connectors/source-smartsheets/source_smartsheets/spec.json

```diff
@@ -7,6 +7,10 @@
     "required": ["access_token", "spreadsheet_id"],
     "additionalProperties": true,
     "properties": {
+      "auth_method": {
+        "type": "string",
+        "const": "oauth2.0"
+      },
```

Comment:
Does the airbyte UI automatically set this to the `const` value? Just want to make sure we're not potentially introducing this requirement to show the auth button without setting it correclty